modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -214,7 +214,6 @@ Currently, the following file formats are supported:
 -   If EXIF information is embedded in the image file, the EXIF orientation will be taken into account
     and thus the image will be rotated accordingly except if the flags @ref IMREAD_IGNORE_ORIENTATION
     or @ref IMREAD_UNCHANGED are passed.
--   Use the IMREAD_UNCHANGED flag to keep the floating point values from PFM image.
```

Comment:
This change is far away from conflict part in this file.
How is it dropped?

That line is a part of 4.x only:
https://github.com/opencv/opencv/blame/4c06a721efb55ab1ec73331c59ac87c3b3f4e043/modules/imgcodecs/include/opencv2/imgcodecs.hpp#L217

BTW, automatic `git merge` works here well.